src/python/pants/pantsd/service/store_gc_service.py

```diff
@@ -8,9 +8,14 @@
 import threading
 import time
 
+from future import standard_library
+
 from pants.pantsd.service.pants_service import PantsService
 
 
+standard_library.install_aliases()
```

Comment:
This should be replaced with `from future.moves import x`. 

I don't see why this is even necessary though, `logging`, `threading`, and `time` are in Py2 and Py3 without major changes and I can't find a relevant library stored from `future.moves`